java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -390,6 +390,24 @@ CompletableFuture<Boolean> pexpireAt(
      */
     CompletableFuture<String> objectEncoding(String key);
 
+    /**
+     * Returns the time in seconds since the last access to the value stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/object-idletime/">redis.io</a> for details.
+     * @param key The <code>key</code> of the object to get the idle time of.
+     * @return If <code>key</code> exists, returns the idle time in seconds of the object at <code>key
```

Comment:
nit: I don't think we need to repeat the description of the entire command in the return statement. like: 
```
     * @return If <code>key</code> exists, returns the idle time in seconds. Otherwise, returns <code>null</code>.
```